pom.xml

```diff
@@ -77,7 +77,7 @@
     -->
     <javadoc.exec.goal>javadoc</javadoc.exec.goal>
 
-    <slf4jVersion>1.7.7</slf4jVersion>
+    <slf4jVersion>1.7.25</slf4jVersion>
```

Comment:
No, this is an optional dep with no impact on the plugin artifact. Ditto core. This dep is very special.